src/test/java/uk/gov/ons/ctp/integration/contactcentresvc/service/impl/CaseServiceImplTest.java

```diff
@@ -100,6 +104,10 @@
 
   @InjectMocks CaseService target = new CaseServiceImpl();
 
+  @Captor private ArgumentCaptor<UUID> individualCaseIdCaptor;
+  @Captor private ArgumentCaptor<CaseContainerDTO> caseCaptor;
+  @Captor private ArgumentCaptor<SurveyLaunchedResponse> surveyLaunchedResponseCaptor;
```

Comment:
I'm very used to defining Captors at the class level. I see them as something different from usual member variables since they are just a convenient device that Mockito needs to do its job in verifying.

For that reason I like to view them as something you just define , ready for using or reusing when you need them.  They won't carry value that you have to hunt down, since they only come to life at the point you use them.

By having them out of the mainline code you have bit less noisy tests.

For that reason I'm very keen to leave this as-is.